tracim/tracim
# Feat/4562 move language into account settings

#4562 
#5452

## Checkpoints



**For developers**

  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified

**For code reviewers**


**For testers**


To test:
- Test if we can still change the language on unlogged pages
- Test if the bug #5452 still happens
- The admin should not be able to change an user language
- The user should be able to change its own language

Note: the follow bug is not linked to this development and it exists already 
- open two tabs
- at tab 1 open a content
- at tab 2 changes the language
- the content on tab 1 will not be changed (but the header will)
